app/src/main/java/fi/bitrite/android/ws/repository/MessageRepository.java

```diff
@@ -165,6 +165,11 @@ public Completable reloadThreads() {
         }).subscribeOn(Schedulers.io());
     }
     public Completable sendMessage(int threadId, String body) {
+        // Format the message as if it was sent from the website.
+        body = "<p>" + body + "</p>";
+        body = body.replaceAll("\n", "<br>");
```

Comment:
Why are you using `replaceAll()` here although you used `replace()` in `Message.java`? Both don’t really need a regular expression and the CharSequence should work in either case?

Reply:
Done.